### PR TITLE
Program migration suggestions

### PR DESCRIPTION
**Pull request by mseaton**
Suggested changes to program migration.

### CODE REVIEW

File: etl/src/main/java/org/pih/hivmigration/etl/sql/ProgramMigrator.groovy [hunk 1]
```diff
@@ -120,19 +120,25 @@ class ProgramMigrator extends SqlMigrator {
...
+            
+            # If no outcome date is readily available, use last visit date + 6 months as a proxy
             UPDATE hivmigration_programs_raw
                 SET outcome_date = DATE(DATE_ADD(max_visit_date, INTERVAL 6 MONTH)) WHERE outcome IS NULL;            
+            
+            # If this results in a future outcome date, then just use the current date
             UPDATE hivmigration_programs_raw
                 SET outcome_date = CURDATE() WHERE outcome_date > CURDATE();
+                
+            # Now, make sure we aren't setting an outcome date if the patient does not have an outcome
```
Comment (mogoodrich) [inline etl/src/main/java/org/pih/hivmigration/etl/sql/ProgramMigrator.groovy:136]: This makes sense... I was wondering why we were setting outcome date for all patients, which seemed wrong?

**Comment by mseaton** [comment]
Thanks @mogoodrich - if you guys are ok with this, can I leave it to you to merge and make sure it doesn't fail?  This hasn't been tested.

**Comment by mogoodrich** [comment]
Sure @mseaton ... @brandones  can you either merge after you review or ping me to remind me to?  Thanks!

**Review by brandones — APPROVED** [review]
Nice, thanks Mike

**Comment by mogoodrich** [comment]
Going ahead and merging this in, thanks @mseaton !

**Pull request merged**